packages/fetch-engine/src/__tests__/download.test.ts

```diff
@@ -10,19 +10,24 @@ import { checkVersionCommand, download, getBinaryName, getVersion } from '../dow
 import { getFiles } from './__utils__/getFiles'
 
 const CURRENT_BINARIES_HASH = enginesVersion
+console.log({ CURRENT_BINARIES_HASH })
 
-const FIXED_BINARIES_HASH = 'a10084a836a379babc008c28b143dc1c7e644453'
+const FIXED_BINARIES_HASH = 'da41d2bb3406da22087b849f0e911199ba4fbf11'
 
 jest.setTimeout(120_000)
 
 describe('download', () => {
   beforeEach(async () => {
     // completely clean up the cache and keep nothing
     await cleanupCache(0)
-    await del(__dirname + '/**/*engine*')
-    await del(__dirname + '/**/prisma-fmt*')
+    // Make sure to not mix forward and backward slashes in the path
+    // or del glob pattern would not work on Windows
+    const dirname = process.platform === 'win32' ? __dirname.split(path.sep).join('/') : __dirname
```

Comment:
Huh how did this even work before?

Reply:
Was not running on Windows before, these changes are needed for the windows runner